### ISSUE TITLE
Input to the Motion Discriminator

### ISSUE THREAD

**Issue by bragilee** [mkocabas/VIBE#132]
Hi authors,

Thanks for your awesome work.

May I have a question about the input to the motion discriminator, why is it 69 dimensions? I suppose it should be 72 dimensions?Did I miss something?

https://github.com/mkocabas/VIBE/blob/8e915c003697e3df281810412037e1054f8b015c/lib/core/loss.py#L122
`end_idx = 75
 start_idx = 6
 pred_motion = total_predict_thetas
 e_motion_disc_loss = self.enc_loss(motion_discriminator(pred_motion[:, :, start_idx:end_idx]))
 e_motion_disc_loss = e_motion_disc_loss * self.d_motion_loss_weight`

Thanks!
